requirements.adoc

```diff
@@ -432,6 +519,8 @@ it is 'scheduled'
 * Verify that a scheduler cannot discontinue the scheduled procedure if
 it is 'completed'
 
+*Judy: Just remember to create this as a role for scheduling since the PT can work as a ascheduler too*
```

Comment:
our roles will get privileges as we think it reflects their work best, but every implementation can configure this however they want.